src/components/UIElements/Bookmarks.vue

```diff
@@ -201,293 +206,303 @@ export default {
-					text-decoration: none;
-					color: var(--font-header);
-					text-overflow: ellipsis;
-					white-space: nowrap;
-					overflow: hidden;
-
-					&:after {
-						display: none;
-					}
-
-					span {
-						font-size: 14px;
-						&:after {
-							display: none;
-						}
-					}
-					
-				}
-
-				.options {
-					position: absolute;
-					pointer-events: none;
-					bottom: 0;
-					left: 7px;
-					width: calc(100% - 7px);
-					display: flex;
-					justify-content: space-between;
-					line-height: 16px;
-					font-size: 16px;
-					color: var(--page-nav-meta);
-
-					span {
-						position: relative;
-						top: 27px;
-						left: 1px;
-						text-overflow: ellipsis;
-						white-space: nowrap;
-						overflow: hidden;
-						font-size: 12px;
-						max-width: 285px;
-						height: 15px;
-					}
-					.systemButton {
-						color: var(--font-default);
-						pointer-events: auto;
-						font-size: 16px;
-						width: 25px;
-						height: 50px;
-						background-color: var(--header-buttonHoverState);
-						transform: translateX(30px);
-						transition: transform 0.03s linear;
-					}
-				}
-			}
-			.editMode {
-				background-color: var(--saves-tabHover);
-
-				display: flex;
-
-				.inputs {
-					padding: 6px 5px;
-				}
-				input {
-					color: var(--font-default);
-					background: var(--header-buttonClickState);
-					border: solid 1px var(--saves-border);
-					font-family: 'Courier New', Courier, monospace;
-					width: 100%;
-
-					&.nameInput {
-						font-weight: bolder;
-						font-size: 14px;
-					}
-					&.urlInput {
-						font-size: 12px;
-					}
-				}
-				.buttons {
-					display: flex;
-					flex-direction: column;
-
-					.systemButton {
-						color: var(--font-default);
-						margin-left: 5px;
-						width: 25px;
-						height: 25px;
-					}
-				}
-
-			}
-		}
-	}
-
-	.bookmarks-enter-active, .bookmarks-leave-active {
-		transition: all .1s;
-	}
-	.bookmarks-enter, .bookmarks-leave-to {
-		transform: translateX(-100%);
-	}
-
-	.bookmark-list-enter, .bookmark-list-leave-to {
-		transform: translateX(-100%);
-	}
-	.bookmark-list-enter-active, .bookmark-list-leave-active {
-		transition: transform 0.1s;
-	}
-	.bookmark-list-move {
-		transition: transform 0.1s;
-	}
-
-	svg {
-		pointer-events: none;
-	}
+  .bookmarkWrapper {
+    z-index: 2;
+    user-select: none;
+    position: fixed;
+    width: 100%;
+    height: calc(100% - var(--headerHeight));
+    overflow-y: hidden;
+    overflow-x: hidden;
+    pointer-events: none;
+
+    color: var(--font-header);
+
+    ::-webkit-scrollbar {
+    width: 6px;
+    }
+    ::-webkit-scrollbar-track {
+      background: var(--saves-border);
+    }
+
+    ::-webkit-scrollbar-thumb {
+      background: var(--saves-scroll);
+      &:hover {
+        background: var(--saves-scrollHover);
+      }
+    }
+
+    .bookmarkShadow {
+      position: fixed;
+      top: 0;
+      left: 330px;
+      width: 100%;
+      height: 100%;
+
+      pointer-events: all;
+    }
```

Comment:
If we're going in this direction we might wanna darken the entire page to indicate it's unclickable. That said, I'm not sure how useful this is as compared to letting people navigate the page while this is open. I figure the kinda person that has a long list of bookmarks might appreciate it if they're trying to cross-reference stuff between pages.